README.md

```diff
@@ -114,45 +114,4 @@ npx source-map-explorer 'packages/code-studio/build/static/js/*.js'
 
 ## Updating Snapshots
 
-Snapshots are used by end-to-end tests to visually verify the output. Sometimes changes are made requiring snapshots to be updated. Since snapshots are platform dependent, you may need to use a docker image to [update snapshots for CI](https://playwright.dev/docs/test-snapshots). You mount the current directory into a docker image and re-run the tests from there.
-
-First start with a clean repo. `node_modules` and some other build output is platform dependent.
-
-```
-npm run clean
-```
-
-Next, start the docker image and open a bash shell inside of it:
-
-```
-docker run --rm --network host -v $(pwd):/work/ -w /work/ -it mcr.microsoft.com/playwright:v1.28.1-focal /bin/bash
-```
-
-Within the docker image shell, install some build tools that are not included with the image:
-
-```
-apt-get update
-apt-get install build-essential --yes
-```
-
-Next, use nvm to install and use the correct version of node:
-
-```
-curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.2/install.sh | bash
-export NVM_DIR="$HOME/.nvm"
-[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
-nvm install
-```
-
-Install npm dependencies and build the production app, pointing to an API running on your machine:
-
-```
-npm ci
-VITE_CORE_API_URL=http://host.docker.internal:10000/jsapi npm run build
-```
-
-Next, run the tests and update the snapshots:
-
-```
-npm run e2e:update-snapshots
-```
+Snapshots are used by end-to-end tests to visually verify the output. Sometimes changes are made requiring snapshots to be updated. Since snapshots are platform dependent, you may need to use a docker image to [update snapshots for CI](https://playwright.dev/docs/test-snapshots). Run `npm run e2e:update-ci-snapshots` to mount the current directory into a docker image and re-run the tests from there. **Note:** You must have [Docker installed](https://docs.docker.com/get-docker/), and `deephaven-core` must already be running on port 10000 on your local machine.
```

Comment:
Update this to mention that you should run local snapshots while creating your tests and only run the docker version once before committing since it will be slow w/ the npm install.

Also after you run the docker version, you need to run `npm install` or `npm ci` locally again to fix your node_modules. See if you can find a way to not require that. Maybe copy the src files into docker, run `npm ci` within docker, and then mount the local `/test` directory to run the tests in Docker (so snapshots go to the right place)